rolling/distribution.yaml

```
@@ -3479,6 +3479,25 @@ repositories:
       url: https://github.com/osrf/osrf_testing_tools_cpp.git
       version: rolling
     status: maintained
+  ouster-ros:
+    doc:
+      type: git
+      url: https://github.com/ouster-lidar/ouster-ros.wiki.git
+      version: master
```

```suggestion
      version: rolling-devel
```

Fix branch name too.